src/app/loginComponents/accounts/controls/controls.component.html

```diff
@@ -1,16 +1,17 @@
 <div class="p-3">
-  <mat-card *ngIf="canChangeUsername$ | async" class="alert alert-danger" role="alert">
+  <!-- <mat-card *ngIf="canChangeUsername$ | async" class="alert alert-danger" role="alert">
```

Comment:
Why is this commented out? It seems like it should still be there. But if it's not supposed to be and it's not  coming back, can you please just delete it.

Reply:
Sorry was planning on working on that tab as well and didn't see the alert on the mockup so temporarily commented it out. Just uncommented on current PR.